packages/workflows/src/handlers/price-list/remove-prices.ts

```diff
@@ -0,0 +1,37 @@
+import { WorkflowArguments } from "../../helper"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPricingModuleService } from "@medusajs/types"
+
+type Result = {
+  deleted: string[]
+}
+
+export async function removePrices({
+  container,
+  data,
+}: WorkflowArguments<{
+  moneyAmountIds: string[]
+}>): Promise<Result | void> {
+  const { moneyAmountIds } = data
+  const pricingService: IPricingModuleService = container.resolve(
+    ModuleRegistrationName.PRICING
+  )
+
+  if (!pricingService) {
+    const logger = container.resolve("logger")
+    logger.warn(
+      `Pricing service not found. You should install the @medusajs/pricing package to use pricing. The 'createPriceList' step will be skipped.`
+    )
+    return void 0
+  }
```

Comment:
question: Why do we only have this check in some handlers?

Reply:
Also the error message is misleading. This is not the createPriceList step.

Reply:
This is legacy from having multiple feature flags, this should really be removed. We have a validation on the loaders to throw an error if this service is not present and v2 flag is on.